website/community/account/de_newchar_3.php

```diff
@@ -111,63 +111,63 @@
 						</td>
 						<td>
 							<?php include_slider( $limits, 'intelligence' ); ?>
 						</td>
 					</tr>
 					<tr>
 						<td>
-							<a title="Die Fähigkeit deines Charakters, seine Sinne einzusetzen. Sie erhöht die Trefferchance im Kampf. Dieses Attribut ist für Bogenschützen wichtig und hilft auch Nahkämpfern, ihr Ziel zu treffen. Wahrnehmung beeinflusst die Lerngeschwindigkeit folgender Fertigkeiten: Distanzwaffen.">Wahrnehmung</a> (<?php echo $limits['minperception'],' - ',$limits['maxperception']; ?>)
+							<a title="Die Fähigkeit deines Charakters, die sechs Sinne zu nutzen. Sie erhöht die Trefferchance im Kampf. Dieses Attribut ist wichtig für Bogenschützen, hilft Nahkämpfern, ihr Ziel zu treffen, und unterstützt Magier beim Zielen. Außerdem verleiht es den Geschmackssinn, der für einen guten Koch, Bäcker und Brauer nötig ist, sowie die scharfe Sicht, um besser zu erkennen, welche Kräuter reif für die Ernte sind. Zudem erleichtert es das Auffinden von Zielen auf Schatzkarten und wird in Zukunft für Druiden wichtig sein. Wahrnehmung beeinflusst die Lerngeschwindigkeit der folgenden Fähigkeiten: Distanzwaffen, Kochen und Backen, Brauen, Kräuterkunde.">Wahrnehmung</a> (<?php echo $limits['minperception'],' - ',$limits['maxperception']; ?>)
 						</td>
 						<td>
 							<?php include_slider( $limits, 'perception' ); ?>
 						</td>
 					</tr>
 					<tr>
 						<td>
-							<a title="Die geistige Widerstandsfähigkeit deines Charakters. Dieses Attribut wird zukünftig für Priester wichtig sein und die Auswirkungen magischer Angriffe abschwächen. Willenskraft beeinflusst die Lerngeschwindigkeit folgender Fertigkeiten: -">Willenskraft</a> (<?php echo $limits['minwillpower'],' - ',$limits['maxwillpower']; ?>)
+							<a title="Die mentale Widerstandskraft deines Charakters. Dieses Attribut erhöht sowohl deine Fähigkeit, Magie zu widerstehen, als auch die Durchschlagskraft der Zauber eines Magiers – ein Wettkampf des stärkeren Willens. Es verleiht dir außerdem die Ausdauer, der Hitze eines Glasbläserofens zu trotzen, sowie dem ständigen Sägestaub, Splittern und Schleifen in der Tischlerei standzuhalten. Zudem hilft es dir, sturer als ein Schaf zu sein, was das Scheren erleichtert. In Zukunft wird es auch für Priester von Bedeutung sein. Willenskraft beeinflusst die Lerngeschwindigkeit der folgenden Fähigkeiten: Magieresistenz, Glasbläserei, Tischlerei, Gerberei und Weberei.">Willenskraft</a> (<?php echo $limits['minwillpower'],' - ',$limits['maxwillpower']; ?>)

```

Comment:
Die mentale Widerstandskraft deines Charakters. Dieses Attribut erhöht sowohl deine Fähigkeit, Magie zu widerstehen, als auch die Durchschlagskraft der Zauber eines Magiers – ein Wettkampf des stärkeren Willens. Es verleiht dir außerdem die Ausdauer, der Hitze eines Glasbläserofens zu trotzen sowie dem ständigen Splittern und Sägestaub als Schreiner standzuhalten. Zudem hilft es dir, sturer als ein Schaf zu sein, was das Scheren erleichtert. In Zukunft wird es auch für Priester von Bedeutung sein. Willenskraft beeinflusst die Lerngeschwindigkeit folgender Fähigkeiten: Magieresistenz, Glasblasen, Schreinern, Gerben und Weben.